docs/en/samples/accelerate_data_accessing_by_hdfs.md

```diff
@@ -185,6 +185,9 @@ metadata:
   name: fluid-hdfs-demo
 spec:
   template:
+    metadata:
+        annotations:
+          fluid.io/dataset-useas-hcfs: hadoop

```

Comment:
Fluid uses labels instead of annotations to indicate the  behavior of webhook. Would you please follow this rule? thanks very much.